src/Modules/CalcPerform.lua

```diff
@@ -3093,6 +3110,65 @@ function calcs.perform(env, avoidCache)
 		end
 	end
 
+	-- Doom Blast (from Impending Doom)
```

Comment:
Is this more appropriately placed in the preDamage calculation for Doom Blast? Many other triggers reside right next to this piece of code, but maybe that's because they are not skill gems.

Reply:
@Paliak or @ProphetLamb are probably better people to evaluate trigger code.

Reply:
I think it's fine to put it here.